oidc_provider/lib/endpoints/token.py

```diff
@@ -42,10 +32,14 @@ def _extract_params(self):
         self.params['client_secret'] = client_secret
         self.params['redirect_uri'] = self.request.POST.get('redirect_uri', '')
         self.params['grant_type'] = self.request.POST.get('grant_type', '')
-        self.params['code'] = self.request.POST.get('code', '')
+        self.params['code'] = self.token_hasher.encode(
+            self.request.POST.get('code', '')
```

Comment:
This is now going to hash an empty string if there is no code. This won't affect how the code works currently at all but would it be better to keep this an empty string if there is no input?

Reply:
Oh yeah, that is definitely silly.

I was just blindly replacing things without considering what should happen.

I will fix it up.